cfg/error_messages.go

```diff
@@ -0,0 +1,59 @@
+package cfg
+
+// This file contains the error messages that get written to the terminal when
+// something goes wrong with the configuration process.
+//
+// As a general rule, if one of these has to be shown the app should then die
+// via os.Exit(1)
+
+import (
+	"fmt"
+
+	"github.com/logrusorgru/aurora"
+)
+
+/* -------------------- Unexported Functions -------------------- */
+
+func displayError(err error) {
+	fmt.Printf("%s %s\n\n", aurora.Red("Error:"), err.Error())
+}
+
+func displayDefaultConfigWriteError(err error) {
+	fmt.Printf("\n%s Could not write the default configuration.\n", aurora.Red("ERROR:"))
```

Comment:
I think this is a little obtuse, considering we throw it when permissions are incorrect. We should explicitly call out permissions here I think?

Reply:
This error will happen if the default `config.yml` cannot be written for any reason into `~/.config/wtf/` on first create.